2013/07/12 11:54:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "controller".' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('controller')
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/ayiitest2/index.php?r=controller
---
2013/07/12 11:58:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "train".' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php:483
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(270): CController->missingAction('train')
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('train')
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('Customer/train')
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/ayiitest2/index.php/Customer/train
---
2013/07/12 12:17:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php:381
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CFormModel.php(39): ModelCode->init()
#6 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('')
#13 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/ayiitest2/index.php/gii/model
HTTP_REFERER=http://localhost/ayiitest2/index.php/gii/model
---
2013/07/12 12:17:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php:381
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CFormModel.php(39): ModelCode->init()
#6 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('')
#13 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/ayiitest2/index.php/gii/model
HTTP_REFERER=http://localhost/ayiitest2/index.php/gii/model
---
2013/08/16 05:10:27 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php:381
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CFormModel.php(39): ModelCode->init()
#6 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('')
#13 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/ayiitest2/index.php/gii/model
HTTP_REFERER=http://localhost/ayiitest2/index.php/gii/module
---
2013/08/16 05:10:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php:381
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CFormModel.php(39): CrudCode->init()
#6 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('')
#13 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#14 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/ayiitest2/index.php/gii/crud
HTTP_REFERER=http://localhost/ayiitest2/index.php/gii/module
---
2013/08/16 14:56:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php:381
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CFormModel.php(39): CrudCode->init()
#6 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('')
#13 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud')
#14 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/ayiitest2/index.php/gii/crud
HTTP_REFERER=http://localhost/ayiitest2/index.php/gii/controller
---
2013/08/16 14:57:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'testdrive'' in D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php:381
Stack trace:
#0 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\08_Tool\xampp\htdocs\ayiitest2\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CModule.php(103): CModule->getComponent('db')
#4 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\generators\model\ModelCode.php(57): CModule->__get('db')
#5 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CFormModel.php(39): ModelCode->init()
#6 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 D:\08_Tool\xampp\htdocs\ayiitest2\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(282): CController->run('')
#13 D:\08_Tool\xampp\htdocs\ayiitest2\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 D:\08_Tool\xampp\htdocs\ayiitest2\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 D:\08_Tool\xampp\htdocs\ayiitest2\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/ayiitest2/index.php/gii/model
HTTP_REFERER=http://localhost/ayiitest2/index.php/gii/module
---
